src/main/groovy/edu/oregonstate/mist/invencycle/resources/BikeResource.groovy

```diff
@@ -66,7 +69,60 @@ class BikeResource extends Resource {
     }
 
     /**
-     * Delete by ID
+     * POST bikes
+     */
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response postBike (@Valid Bike newBike) {
+        Response returnResponse
+        try {
+            bikeDAO.postBrakeFront(newBike.brake_make_front,
```

Comment:
hmm, I'm wondering if it would be better to have a method in BikeDAO that you can call. For example:
`bikeDAO.addBike(newBike)`

That method would then take care of calling all the internal bikeDAO methods. It would make this method much shorter and easier to read. It should also help with the PUT when you get to that piece as well.